tests/e2e/specs/create_and_remove-existing-menu-test.js

```diff
@@ -0,0 +1,59 @@
+import { loginUser, visitAdminPage,activateTheme } from '@wordpress/e2e-test-utils';
```

Comment:
It might make sense to rename the test file to something like create-remove-menu.test.js or something similar. The idea is to keep consistency with existing tests files names.

Also, I'd move this test to a "Menu" folder as we might have others related tests in the future.